### PR TITLE
feat(movies): add controller and tests

### PR DESCRIPTION
**Pull request by jekatlob**
Why: wire things up end-to-end

What: add a controller so the create endpoint will actually persist the contents of the request to the DB

Details: still trying to remember how to git so once again the commit history is messier than I'd like

### CODE REVIEW

File: lib/plugins/features/movies/controller.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+exports.create = async (payload) => {
+  const movie = await new Movie().save(payload);
+
+  return new Movie({ id: movie.id }).fetch();
```
Comment (sjl2) [inline lib/plugins/features/movies/controller.js]: nit: can you add `{ require: true}` to the fetch options?

File: test/plugins/features/movies/controller.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  describe('create', () => {
+
+    it('creates a movie', async () => {
+      const payload = { title: 'WALL-E' };
+
+      const movie = await Controller.create(payload);
+
+      expect(movie.get('title')).to.eql(payload.title);
+    });
```
Comment (sjl2) [inline test/plugins/features/movies/controller.test.js]: nit: can you add an expect for checking the DB to make sure it was written? We've recently moved to doing that after an incident.

File: test/plugins/features/movies/index.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Movies = require('../../../../lib/server');
+
+describe('movies integration', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
+      return Movies.inject({
```
Comment (sjl2) [inline test/plugins/features/movies/index.test.js]: super nit: can you make this `async` await?

**Comment by jekatlob** [comment]
After talking to @sjl2 IRL, we agreed that leaving the nits unchanged for now is fine (we'll note them for future modification to the tutorial).

**Pull request merged**